### ISSUE TITLE
Add support for 0.11.14 and continuously support all stable TF releases 

### ISSUE THREAD

**Issue by Mykolaichenko** [hashicorp/terraform-github-actions#42]
Hi guys, 

Please add support for: 
https://releases.hashicorp.com/terraform/0.11.14/

And I think we need to create scheduled release process for every Terraform version. 

WDYT?

**Comment by sudomateo** [comment]
I'm in the process of working on a way to pass the Terraform version to be used as an input. Stay tuned!

**Comment by jeffatgametime** [comment]
how's this going? we're running into an issue here

we recently imported a lot of infrastructure using `v0.12.10` but the terraform version in the action from hashicorp is `v0.12.9`. any help would be appreciated.

 ```
Error: Error loading state: state snapshot was created by Terraform v0.12.10, which is newer than current v0.12.9; upgrade to Terraform v0.12.10 or greater to work with this state
```

**Comment by sudomateo** [comment]
I'm working on a refactor that will allow this. Meanwhile I'll be able to cut a new release for 0.12.10.

**Comment by sudomateo** [comment]
Should be fixed in #88.
